metadata.json

```diff
@@ -15,10 +15,6 @@
     {
       "name": "puppetlabs/stdlib",
       "version_requirement": ">= 4.20.0 < 5.0.0"
-    },
-    {
-      "name": "camptocamp/systemd",
```

Comment:
It becomes a 'soft' dependency.  If somebody wants to take this PR on (maybe add a test or two for `$use_camptocamp_systemd => false`), then the README should probably get a sentence explaining how this works.

Reply:
I like removal of hard dependencies